components/listview/overview.md

```diff
@@ -66,18 +69,29 @@ the rest of the features the component provides further in this article *@
 </style>
 ````
 
->caption The result from the code snippet above
+## Templates
+
+You fully control the rendering of the items and their layout. You can add header, footer, and a template for editing items. [Read more about the Blazor ListView templates]({%slug listview-templates%}).
+
+## Editing
+
+The ListView component allows you to define a template for an item in edit mode and add command buttons for the user to edit, delete and insert items. [Read more about the Blazor ListView editing]({%slug listview-editing%}).
+
+## Paging
 
-![ListView basic example](images/listview-first-look.png)
+You can enable automatic paging of the provided data so the user has less scrolling to do and the list can fit better in your layout. [Read more about the Blazor ListView paging]({%slug listview-paging%}).
 
->caption The ListView provides the following features:
+## Refresh Data
 
-* [Paging]({%slug listview-paging%}) - you can enable automatic paging of the provided data so the user has less scrolling to do and the list can fit better in your layout
-* [Templates]({%slug listview-templates%}) - you fully control the rendering of the items and their layout. You can add header, footer, and a template for editing items.
-* [Editing]({%slug listview-editing%}) - you can define a template for an item in edit mode and add command buttons for the user to edit, delete and insert items.
-* [Events]({%slug listview-events%}) - the CRUD operations happen through dedicated events, and there is also an option for [load-on-demand for the data]({%slug listview-manual-operations%}), which provides you with full control over the data operations and not only the rendering.
+The ListView allows you to refresh its data manually so the component can react to changes in the collection. [Read more about the Blazor ListView data refresh]({%slug listview-refresh-data%}).
 
->caption Component namespace and reference
+## Events
+
+The CRUD operations happen through dedicated events, and there is also an option for [load-on-demand for the data]({%slug listview-manual-operations%}), which provides you with full control over the data operations and not only the rendering. [Read more about the Blazor ListView events]({%slug listview-events%}).
+
+## ListView Reference
```

Comment:
The Reference section should demonstrate actual reference usage. Usually these are methods. If a component doesn't have methods, the section is not necessary.